### ISSUE TITLE
This quiz has 4 questions and only 1 question has to deal with the topic we are on currently. 

### ISSUE THREAD

**Issue by soojkim0106** [learn-co-curriculum/python-p3-quiz-data-structures#7]
### Canvas Link

https://learning.flatironschool.com/courses/7237/quizzes/55673

### Concern

This quiz is not working as intended.

**Comment by bencavins** [comment]
Pinging this, it is confusing current phase 3 students. I think this quiz should not be marked required until it is fixed.

**Comment by rhythms06** [comment]
I agree with @bencavins, this quiz should not be required until it more appropriately covers the topic of data structures.

**Comment by a-JAVA-chip-frappuccino** [comment]
I'm agreeing as well! I think the first question is very good, and the third is also relatively decent (given DSAs are discussed somewhat throughout), but I'd definitely not make this gating at this point in the lesson.
